lang/py/src/avro/schema.py

```diff
@@ -815,21 +896,38 @@ def make_avsc_object(json_data, names=None):
     logical_type = None
     if 'logicalType' in json_data:
       logical_type = json_data.get('logicalType')
-      if logical_type != 'decimal':
+      if logical_type not in [constants.DATE, constants.TIME_MILLIS, constants.TIME_MICROS, constants.TIMESTAMP_MILLIS, constants.TIMESTAMP_MICROS, constants.DECIMAL]:
```

Comment:
maybe worth making a constant tuple that includes all the supported logical types like `PRIMITIVE_TYPES`, `NAMED_TYPES`, etc.?